### PR TITLE
Added card_uri and updated resource_property to match documentation

### PR DESCRIPTION
**Pull request by carmenjyuen**
Except for Lead Gen and App Download card

### CODE REVIEW

**Comment by carmenjyuen** [comment]
Going to make some more edits

File: twitter_ads/creative.py [hunk 1]
```diff
@@ -79,18 +79,22 @@ class Video(Resource, Persistence):
 
 # video properties
 # read-only
+resource_property(Video, 'aspect_ratio', readonly=True)
+resource_property(Video, 'created_at', readonly=True, transform=TRANSFORM.TIME)
+resource_property(Video, 'deleted', readonly=True, transform=TRANSFORM.BOOL)
+resource_property(Video, 'description')
+resource_property(Video, 'duration', readonly=True)
```
Comment (juanshishido) [inline twitter_ads/creative.py]: Let's `transform=TRANSFORM.INT` here

File: twitter_ads/creative.py [hunk 2]
```diff
@@ -79,18 +79,22 @@ class Video(Resource, Persistence):
...
 resource_property(Video, 'id', readonly=True)
-resource_property(Video, 'tweeted', readonly=True, transform=TRANSFORM.BOOL)
+resource_property(Video, 'media_key', readonly=True)
+resource_property(Video, 'poster_url', readonly=True)
+resource_property(Video, 'preview_url', readonly=True)
 resource_property(Video, 'ready_to_tweet', readonly=True, transform=TRANSFORM.BOOL)
-resource_property(Video, 'duration', readonly=True)
 resource_property(Video, 'reasons_not_servable', readonly=True)
```
Comment (juanshishido) [inline twitter_ads/creative.py]: Let's `transform=TRANSFORM.LIST` here

File: twitter_ads/creative.py [hunk 3]
```diff
@@ -104,17 +108,16 @@ class AccountMedia(Resource, Persistence):
 
 # video properties
```
Comment (juanshishido) [inline twitter_ads/creative.py]: Let's update the comment here

File: twitter_ads/creative.py [hunk 4]
```diff
@@ -79,18 +79,22 @@ class Video(Resource, Persistence):
...
 # read-only
+resource_property(Video, 'aspect_ratio', readonly=True)
+resource_property(Video, 'created_at', readonly=True, transform=TRANSFORM.TIME)
+resource_property(Video, 'deleted', readonly=True, transform=TRANSFORM.BOOL)
+resource_property(Video, 'description')
```
Comment (juanshishido) [inline twitter_ads/creative.py]: Since this is writable, let's move it down to that section

File: twitter_ads/creative.py [hunk 5]
```diff
@@ -104,17 +108,16 @@ class AccountMedia(Resource, Persistence):
 
 # video properties
 # read-only
+resource_property(AccountMedia, 'created_at', readonly=True, transform=TRANSFORM.TIME)
+resource_property(AccountMedia, 'deleted', readonly=True, transform=TRANSFORM.BOOL)
+resource_property(AccountMedia, 'duration', readonly=True)
```
Comment (juanshishido) [inline twitter_ads/creative.py]: Not sure this is available for this entity

File: twitter_ads/creative.py [hunk 6]
```diff
@@ -127,17 +130,17 @@ class MediaCreative(Resource, Persistence):
 
 # video properties
```
Comment (juanshishido) [inline twitter_ads/creative.py]: Let's update the comment here

File: twitter_ads/creative.py [hunk 7]
```diff
@@ -190,8 +200,6 @@ class VideoWebsiteCard(Resource, Persistence):
 resource_property(VideoWebsiteCard, 'video_poster_width', readonly=True)
 resource_property(VideoWebsiteCard, 'video_url', readonly=True)
 resource_property(VideoWebsiteCard, 'video_width', readonly=True)
-resource_property(VideoWebsiteCard, 'website_dest_url', readonly=True)
```
Comment (juanshishido) [inline twitter_ads/creative.py]: It looks like `website_dest_url` and `website_display_url` *are* returned in the response. The documentation just doesn't reflect that yet. Let's add these back in

File: twitter_ads/creative.py [hunk 8]
```diff
@@ -297,24 +310,29 @@ class VideoAppDownloadCard(Resource, Persistence):
...
+resource_property(VideoAppDownloadCard, 'app_cta')
+resource_property(VideoAppDownloadCard, 'image_media_id')
 resource_property(VideoAppDownloadCard, 'ipad_app_id')
 resource_property(VideoAppDownloadCard, 'ipad_deep_link')
+resource_property(VideoAppDownloadCard, 'iphone_app_id')
+resource_property(VideoAppDownloadCard, 'iphone_deep_link')
 resource_property(VideoAppDownloadCard, 'googleplay_app_id')
 resource_property(VideoAppDownloadCard, 'googleplay_deep_link')
-resource_property(VideoAppDownloadCard, 'app_cta')
-resource_property(VideoAppDownloadCard, 'image_media_id')
```
Comment (juanshishido) [inline twitter_ads/creative.py]: `image_media_id` *should* be included

File: twitter_ads/creative.py [hunk 9]
```diff
@@ -328,21 +346,29 @@ class ImageConversationCard(Resource, Persistence):
...
+resource_property(ImageConversationCard, 'fouth_cta_tweet')
+resource_property(ImageConversationCard, 'image_media_id')
 resource_property(ImageConversationCard, 'first_cta')
 resource_property(ImageConversationCard, 'first_cta_tweet')
+resource_property(ImageConversationCard, 'name')
 resource_property(ImageConversationCard, 'second_cta')
 resource_property(ImageConversationCard, 'second_cta_tweet')
 resource_property(ImageConversationCard, 'thank_you_text')
 resource_property(ImageConversationCard, 'thank_you_url')
-resource_property(ImageConversationCard, 'image_media_id')
```
Comment (juanshishido) [inline twitter_ads/creative.py]: Include `image_media_id`

**Comment by juanshishido** [comment]
Thank you for this contribution, @carmenjyuen!

**Pull request merged**